pom.xml

```diff
@@ -277,6 +282,36 @@
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
```

Comment:
This artifact does not seem to exist to versions 1.1.2 and 1.2.1 (currently still listed as supported). I guess it's OK since this is for the tests only but is there a way to make it work for these as well? I'm not saying we should replicate the test support classes.